src/routes/articles/update-article.e2e.test.js

```diff
@@ -0,0 +1,196 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const { fixtures } = require('../../../test/utils')
+
+describe('PATCH /articles/:id', () => {
+	let user
+	let accessToken
+	let article
+	let updatedInfo = {}
+	beforeEach(async ()=> {
```

Comment:
we're sure we need these actions before each test case?

Reply:
not exactly